features/git-town/configuration/configuration_reset.feature

```diff
@@ -0,0 +1,36 @@
+Feature: resetting the configuration
+
+  As a user wishing to remove all Git Town settings from a repository
+  I want to be able to cleanly reset the Git Town configuration with a single command
+  So that I can continue working on actual work and stay productive
```

Comment:
Okay, lets go with uninstalling Git Town configuration as the user value.

> As a user no longer using Git Town on a repository
> I want to be able to cleanly remove all Git Town configuration from my Git repo
> So that my repository is left in a clean state after the uninstallation.